# Translations template for PROJECT.
# Copyright (C) 2019 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
#,
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2019-03-11 09:48+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

#: apps/desks.py:203
msgid "Cannot update Desk Type as there are article(s) referenced by the Desk."
msgstr ""

#: apps/desks.py:226
msgid "Cannot delete desk as it is assigned as default desk to user(s)."
msgstr "DE - Cannot delete desk as it is assigned as default desk to user(s)."

#: apps/desks.py:237
msgid "Cannot delete desk as routing scheme(s) are associated with the desk"
msgstr ""

#: apps/desks.py:250
msgid ""
"Cannot delete desk as it has article(s) or referenced by versions of the "
"article(s)."
msgstr ""

#: apps/preferences.py:242 apps/preferences.py:254
msgid "Invalid preference: {preference}"
msgstr ""

#: apps/stages.py:164
msgid "Cannot delete a Working Stage."
msgstr ""

#: apps/stages.py:169
msgid "Cannot delete a Incoming Stage."
msgstr ""

#: apps/stages.py:175
msgid ""
"Cannot delete stage as it has article(s) or referenced by versions of the"
" article(s)."
msgstr ""

#: apps/stages.py:182
msgid "Stage is referred by Ingest Routing Schemes : {rule_names}"
msgstr ""

#: apps/stages.py:202
msgid "Must have one working stage in a desk"
msgstr ""

#: apps/stages.py:210
msgid "Must have one incoming stage in a desk"
msgstr ""

#: apps/archive/common.py:542 apps/tasks.py:84
msgid "Invalid desk identifier {desk_id}"
msgstr ""

#: apps/archive/common.py:548 apps/tasks.py:94
msgid "Invalid stage identifier {stage_id}"
msgstr ""

#: apps/tasks.py:127
msgid "Error:{exception} in {rule_type} rule:{rule} for stage:{stage}"
msgstr ""

#: apps/archive/archive.py:183 apps/archive/archive.py:772
msgid "Package doesn't support Public Service Announcements"
msgstr ""

#: apps/archive/archive.py:356 apps/archive/archive.py:766
#: apps/duplication/archive_duplication.py:131
#: apps/item_autosave/components/item_autosave.py:35
msgid "The item was locked by another user"
msgstr ""

#: apps/archive/archive.py:392 apps/archived/archived.py:187
msgid "User does not have permissions to read the item."
msgstr ""

#: apps/archive/archive.py:407
msgid "Invalid version {old_version}"
msgstr ""

#: apps/archive/archive.py:411
msgid "Invalid item id {item_id}"
msgstr ""

#: apps/archive/archive.py:415
msgid "Invalid last version {last_version}"
msgstr ""

#: apps/archive/archive.py:690
msgid "An item can't have both Publish Schedule and Embargo"
msgstr ""

#: apps/archive/archive.py:695
msgid "Embargo cannot be earlier than now"
msgstr ""

#: apps/archive/archive.py:698
msgid "Rewrites doesn't support Embargo"
msgstr ""

#: apps/archive/archive.py:701
msgid "Invalid Embargo"
msgstr ""

#: apps/archive/archive.py:705
msgid "A Package doesn't support Embargo"
msgstr ""

#: apps/archive/archive.py:769
msgid "Item isn't in a valid state to be updated."
msgstr ""

#: apps/archive/archive.py:777
msgid "Unauthorized to modify Unique Name"
msgstr ""

#: apps/archive/archive.py:782
msgid "Cannot change the genre for broadcast content."
msgstr ""

#: apps/archive/archive.py:787
msgid ""
"This item is in a package and it needs to be removed before the item can "
"be scheduled!"
msgstr ""

#: apps/archive/archive.py:813
msgid "Duplicate category codes are not allowed"
msgstr ""

#: apps/archive/archive.py:818
msgid "Duplicate subjects are not allowed"
msgstr ""

#: apps/archive/archive.py:956
msgid "Client and server etags don't match"
msgstr ""

#: apps/archive/archive.py:958
msgid "Request for Auto-save must have _id"
msgstr ""

#: apps/archive/archive_link.py:58 apps/archive/archive_rewrite.py:274
msgid "Only updates can be unlinked!"
msgstr ""

#: apps/archive/archive_rewrite.py:98
msgid "Cannot find the article"
msgstr ""

#: apps/archive/archive_rewrite.py:101
msgid "Rewrite of an Item having embargo isn't possible"
msgstr ""

#: apps/archive/archive_rewrite.py:104
msgid "Event id does not exist"
msgstr ""

#: apps/archive/archive_rewrite.py:107
msgid "Article has been rewritten before !"
msgstr ""

#: apps/archive/archive_rewrite.py:114
msgid "Rewrite is not published. Cannot rewrite the story again."
msgstr ""

#: apps/archive/archive_rewrite.py:119
msgid "Rewrite story has been used as update before !"
msgstr ""

#: apps/archive/archive_rewrite.py:130
msgid "Rewrite story can only be text or pre-formatted !"
msgstr ""

#: apps/archive/archive_rewrite.py:134
msgid "Broadcast cannot be a update story !"
msgstr ""

#: apps/archive/archive_rewrite.py:137
msgid "Rewrite item content profile does not match with Original item."
msgstr ""

#: apps/archive/archive_spike.py:95
msgid "The item \"{slugline}\" is in a package"
msgstr ""

#: apps/archive/archive_spike.py:96
msgid " it needs to be removed before the item can be spiked"
msgstr ""

#: apps/archive/common.py:435
msgid "Document not found in archive collection"
msgstr ""

#: apps/archive/common.py:624 apps/archive/common.py:626
msgid "Schedule date is not recognized"
msgstr ""

#: apps/archive/common.py:628
msgid "Schedule cannot be earlier than now"
msgstr ""

#: apps/archive_broadcast/broadcast.py:85
msgid "Cannot find the {genre} genre."
msgstr ""

#: apps/archive_broadcast/broadcast.py:115
msgid "Cannot find the requested item id."
msgstr ""

#: apps/archive_broadcast/broadcast.py:118
msgid "Invalid content type."
msgstr ""

#: apps/archive_broadcast/broadcast.py:121
msgid "Invalid content state."
msgstr ""

#: apps/archived/archived.py:126
msgid "Only Text articles are allowed to be Killed in Archived repo"
msgstr ""

#: apps/archived/archived.py:129
msgid "Killing of Broadcast Items isn't allowed in Archived repo"
msgstr ""

#: apps/archived/archived.py:133
msgid ""
"Can't kill as this article acts as a Master Story for existing "
"broadcast(s)"
msgstr ""

#: apps/archived/archived.py:136
msgid "Can't Kill as article is still available in production"
msgstr ""

#: apps/archived/archived.py:139
msgid "Can't kill as article is part of a Package"
msgstr ""

#: apps/archived/archived.py:144
msgid "Can't Kill as the Digital Story is still available in production"
msgstr ""

#: apps/archived/archived.py:150
msgid "Digital Story of the article not found in Archived repo"
msgstr ""

#: apps/archived/archived.py:154
msgid "Can't kill as Digital Story is part of a Package"
msgstr ""

#: apps/archived/archived.py:159
msgid "Can't Kill as Take(s) are still available in production"
msgstr ""

#: apps/archived/archived.py:163
msgid "One of Take(s) not found in Archived repo"
msgstr ""

#: apps/archived/archived.py:166
msgid "Can't kill as one of Take(s) is part of a Package"
msgstr ""

#: apps/auth/__init__.py:64
msgid "Invalid user."
msgstr ""

#: apps/auth/db/reset_password.py:80
msgid "Either key:password or email must be provided"
msgstr ""

#: apps/auth/db/reset_password.py:98
msgid "Invalid email"
msgstr ""

#: apps/auth/db/reset_password.py:102
msgid "User not enabled"
msgstr ""

#: apps/auth/db/reset_password.py:106 apps/auth/db/reset_password.py:123
msgid "User not active"
msgstr ""

#: apps/comments/comments.py:84
msgid "Commenting on behalf of someone else is prohibited."
msgstr ""

#: apps/content_filters/content_filter/content_filter_service.py:39
msgid "Content filter has been referenced by subscriber(s) {references}"
msgstr ""

#: apps/content_filters/content_filter/content_filter_service.py:47
msgid "Content filter has been referenced by routing scheme(s) {references}"
msgstr ""

#: apps/content_filters/content_filter/content_filter_service.py:55
msgid "Content filter has been referenced in {references})"
msgstr ""

#: apps/content_filters/content_filter/content_filter_service.py:125
msgid "Circular dependency error in content filters:{filter}"
msgstr ""

#: apps/content_filters/content_filter/content_filter_test.py:46
msgid "Content filter not found"
msgstr ""

#: apps/content_filters/content_filter/content_filter_test.py:54
#: apps/products/product_test.py:49 apps/publish/formatters/service.py:66
msgid "Article not found!"
msgstr ""

#: apps/content_filters/content_filter/content_filter_test.py:59
msgid "Error in testing article: {error}"
msgstr ""

#: apps/content_filters/content_filter/content_filter_test.py:73
msgid "Error in testing archive: {error}"
msgstr ""

#: apps/content_filters/filter_condition/filter_condition_field.py:51
msgid "Invalid filter conditions field {field}"
msgstr ""

#: apps/content_filters/filter_condition/filter_condition_service.py:35
msgid "Filter condition has been referenced in content filter: {references}"
msgstr ""

#: apps/content_filters/filter_condition/filter_condition_service.py:49
msgid "Filter condition:{condition} has unidentified field: {field}"
msgstr ""

#: apps/content_filters/filter_condition/filter_condition_service.py:53
msgid "Filter condition:{condition} has unidentified operator: {operator}"
msgstr ""

#: apps/content_filters/filter_condition/filter_condition_service.py:70
msgid "Filter condition:{condition} has identical settings"
msgstr ""

#: apps/content_types/content_types.py:130
msgid "Vocabulary {vocabulary} is used in {count} content type(s)"
msgstr ""

#: apps/content_types/content_types.py:150
msgid ""
"Cannot disable content profile as following templates are referencing: "
"{templates}"
msgstr ""

#: apps/content_types/content_types.py:161
msgid "Cannot disable content profile as following desks are referencing: {desks}"
msgstr ""

#: apps/dictionaries/service.py:142
msgid "A text dictionary file is required"
msgstr ""

#: apps/dictionaries/service.py:162
msgid "The dictionary already exists"
msgstr ""

#: apps/dictionaries/service.py:186
msgid "User is required for the abbreviations dictionary."
msgstr ""

#: apps/dictionaries/service.py:190
msgid "The dictionary type cannot be changed."
msgstr ""

#: apps/duplication/archive_copy.py:54
#: apps/duplication/archive_duplication.py:120
#: apps/duplication/archive_move.py:106
#: apps/duplication/archive_translate.py:66
msgid "Fail to found item with guid: {guid}"
msgstr ""

#: apps/duplication/archive_copy.py:59
msgid "Copy is not allowed on items in a desk."
msgstr ""

#: apps/duplication/archive_fetch.py:74
msgid "Fail to found ingest item with _id: {id}"
msgstr ""

#: apps/duplication/archive_move.py:72
msgid "Item is locked for move by another user."
msgstr ""

#: apps/duplication/archive_move.py:153
msgid "Move is not allowed within the same stage."
msgstr ""

#: apps/export/service.py:63
msgid "Error creating export zip file. Try again please."
msgstr ""

#: apps/export/service.py:76
msgid "No items to export."
msgstr ""

#: apps/export/service.py:81
msgid "Formatter not found for requested format type."
msgstr ""

#: apps/io/search_ingest.py:66
msgid "Destination desk cannot be empty."
msgstr ""

#: apps/io/search_ingest.py:108
msgid "provider not found source={source}"
msgstr ""

#: apps/io/feeding_services/reuters.py:148
msgid "Not found {payload}"
msgstr ""

#: apps/item_autosave/components/item_autosave.py:31
msgid "Invalid item identifier"
msgstr ""

#: apps/item_lock/components/item_lock.py:73
msgid "Item is locked by another user."
msgstr ""

#: apps/item_lock/components/item_lock.py:118
msgid "Item is not locked."
msgstr ""

#: apps/keywords/alchemy.py:27
msgid "AlchemyAPI key is not set"
msgstr ""

#: apps/keywords/alchemy.py:41
msgid "Fail to connect to Alchemy service"
msgstr ""

#: apps/keywords/alchemy.py:47
msgid "Fail to parse the response from Alchemy service"
msgstr ""

#: apps/keywords/service.py:25
msgid "Not set a keywords provider"
msgstr ""

#: apps/ldap/ldap.py:125
msgid "User does not belong to security Group or could not find the user profile."
msgstr ""

#: apps/ldap/ldap.py:174
msgid "No user has been found in AD"
msgstr ""

#: apps/ldap/ldap.py:201 apps/ldap/ldap.py:208
msgid "Invalid Credentials."
msgstr ""

#: apps/ldap/ldap.py:211
msgid "User already exists in the system."
msgstr ""

#: apps/packages/package_service.py:131
msgid "Root group is missing."
msgstr ""

#: apps/packages/package_service.py:136
msgid "Only one root group is allowed."
msgstr ""

#: apps/packages/package_service.py:144
msgid "Group is missing id."
msgstr ""

#: apps/packages/package_service.py:155
msgid "{id} group is added multiple times."
msgstr ""

#: apps/packages/package_service.py:160
msgid ""
"The number of groups and of referenced groups in the root group do not "
"match."
msgstr ""

#: apps/packages/package_service.py:165
msgid "Not all groups are referenced in the root group."
msgstr ""

#: apps/packages/package_service.py:210
msgid "Package contains empty ResidRef!"
msgstr ""

#: apps/packages/package_service.py:215
msgid "Invalid item reference: {reference}"
msgstr ""

#: apps/packages/package_service.py:245
msgid "Trying to self reference as an association."
msgstr ""

#: apps/packages/package_service.py:251
msgid "Content associated multiple times"
msgstr ""

#: apps/packages/package_service.py:257
msgid "Trying to create a circular reference to: {item_id}"
msgstr ""

#: apps/packages/package_service.py:430
msgid "Package can't have item which has embargo. "
msgstr ""

#: apps/packages/package_service.py:431
msgid "Slugline/Unique Name of the item having embargo: {slugline}/{unique}"
msgstr ""

#: apps/products/product_test.py:43
msgid "Article id cannot be empty!"
msgstr ""

#: apps/products/product_test.py:56
msgid "Error in testing article: {exception}"
msgstr ""

#: apps/products/service.py:31
msgid "Product is used by the subscriber(s): {names}"
msgstr ""

#: apps/products/service.py:47
msgid "Product is used for API publishing for the subscriber(s): {subscribers}"
msgstr ""

#: apps/products/service.py:54
msgid "Product is used for direct publishing for the subscriber(s): {subscribers}"
msgstr ""

#: apps/publish/published_item.py:177
msgid "Invalid state ({state}) for the Published item."
msgstr ""

#: apps/publish/published_item.py:232 apps/publish/published_item.py:244
#: apps/publish/published_item.py:256
msgid ""
"Deleting a published item has a workflow which is implemented in "
"remove_expired()."
msgstr ""

#: apps/publish/content/common.py:181
#: apps/publish/enqueue/enqueue_service.py:197
msgid "Key is missing on article to be published: {exception}"
msgstr ""

#: apps/publish/content/common.py:185
msgid "Failed to publish the item: {id}"
msgstr ""

#: apps/publish/content/common.py:225
msgid "Embargo can't be set after publishing"
msgstr ""

#: apps/publish/content/common.py:229
msgid "Dateline can't be modified on kill or take down"
msgstr ""

#: apps/publish/content/common.py:234
msgid "Cannot publish the story after Update is published.!"
msgstr ""

#: apps/publish/content/common.py:264 apps/publish/content/common.py:323
#: apps/publish/enqueue/enqueue_service.py:121
msgid "Corrected package cannot be empty!"
msgstr ""

#: apps/publish/content/common.py:268
msgid "Cannot publish an item which is marked as Not for Publication"
msgstr ""

#: apps/publish/content/common.py:272
msgid "Can't {operation} as item state is {state}"
msgstr ""

#: apps/publish/content/common.py:274
msgid ""
"Can't {operation} as either package state or one of the items state is "
"{state}"
msgstr ""

#: apps/publish/content/common.py:316 apps/publish/content/publish.py:41
msgid "Empty package cannot be published!"
msgstr ""

#: apps/publish/content/common.py:336
msgid "Package item with id: {guid} does not exist."
msgstr ""

#: apps/publish/content/kill.py:56
msgid ""
"This item is in a package. It needs to be removed before the item can be "
"killed"
msgstr ""

#: apps/publish/content/published_package_items.py:61
msgid "Invalid package identifier"
msgstr ""

#: apps/publish/content/published_package_items.py:63
msgid "Package was not published"
msgstr ""

#: apps/publish/content/published_package_items.py:70
msgid "Invalid item identifier  {item_id}"
msgstr ""

#: apps/publish/content/published_package_items.py:75
msgid "Circular reference in item {item_id}"
msgstr ""

#: apps/publish/content/resend.py:59
msgid "No subscribers selected!"
msgstr ""

#: apps/publish/content/resend.py:65
msgid "No active subscribers found!"
msgstr ""

#: apps/publish/content/resend.py:70
msgid "Only wire subscribers can receive broadcast stories!"
msgstr ""

#: apps/publish/content/resend.py:78
msgid "Story couldn't be found!"
msgstr ""

#: apps/publish/content/resend.py:82
msgid "Only text stories can be resent!"
msgstr ""

#: apps/publish/content/resend.py:87
msgid "Only published, corrected or killed stories can be resent!"
msgstr ""

#: apps/publish/content/resend.py:91
msgid "Please use the newest version {version} to resend!"
msgstr ""

#: apps/publish/content/resend.py:96
msgid "Updated story cannot be resent!"
msgstr ""

#: apps/publish/enqueue/enqueue_service.py:131
msgid "Package item with id: {guid} has not been published."
msgstr ""

#: apps/publish/enqueue/enqueue_service.py:201
msgid "Failed to publish the item: {exception}"
msgstr ""

#: apps/publish/enqueue/enqueue_service.py:511
msgid "Can not find package {package} published item {item}"
msgstr ""

#: apps/publish/formatters/service.py:54
msgid "Formatter name not found"
msgstr ""

#: apps/publish/formatters/service.py:59
msgid "Formatter not found"
msgstr ""

#: apps/publish/formatters/service.py:80
msgid "Error in formatting article: {exception}"
msgstr ""

#: apps/rules/routing_rules.py:201
msgid "Routing scheme is applied to channel(s). It cannot be deleted."
msgstr ""

#: apps/rules/routing_rules.py:293
msgid "A Routing Scheme must have at least one Rule"
msgstr ""

#: apps/rules/routing_rules.py:300
msgid "A routing rule has invalid fields {fields}"
msgstr ""

#: apps/rules/routing_rules.py:306
msgid "A routing rule must have a name"
msgstr ""

#: apps/rules/routing_rules.py:309
msgid "A routing rule must have actions"
msgstr ""

#: apps/rules/routing_rules.py:328
msgid "Schedule when defined can't be empty."
msgstr ""

#: apps/rules/routing_rules.py:332
msgid "Invalid values for day of week."
msgstr ""

#: apps/rules/routing_rules.py:338
msgid "Invalid value for from time."
msgstr ""

#: apps/rules/routing_rules.py:346
msgid "Invalid value for hour_of_day_to (expected %H:%M:%S)."
msgstr ""

#: apps/rules/routing_rules.py:350
msgid "From time should be less than To Time."
msgstr ""

#: apps/rules/routing_rules.py:356
msgid "Unknown time zone {time_zone}"
msgstr ""

#: apps/rules/routing_rules.py:369
msgid "Rule Names must be unique within a scheme"
msgstr ""

#: apps/rules/rule_sets.py:54
msgid "Cannot delete Rule set as it's associated with channel(s)."
msgstr ""

#: apps/saved_searches/saved_searches.py:160
msgid "User's subscriptions are not allowed on create"
msgstr ""

#: apps/saved_searches/saved_searches.py:185
msgid "Unauthorized to modify subscriptions."
msgstr ""

#: apps/saved_searches/saved_searches.py:202
msgid "Unauthorized to modify other users' subscriptions."
msgstr ""

#: apps/saved_searches/saved_searches.py:266
msgid "Search cannot be saved without a filter!"
msgstr ""

#: apps/saved_searches/saved_searches.py:287
msgid "Fail to validate the filter against {index}."
msgstr ""

#: apps/saved_searches/saved_searches.py:306
msgid "Unauthorized to modify other user's local search."
msgstr ""

#: apps/saved_searches/saved_searches.py:308
msgid "Unauthorized to modify global search."
msgstr ""

#: apps/saved_searches/saved_searches.py:333
msgid "Invalid Saved Search"
msgstr ""

#: apps/search_providers/registry.py:28
msgid "fetch_enpoint must be a string"
msgstr ""

#: apps/search_providers/registry.py:33
msgid "You have to specify either fetch_endpoint or provider_class."
msgstr ""

#: apps/search_providers/registry.py:41
msgid ""
"A Search Provider for the fetch endpoint: {endpoint} exists with name: "
"{name}"
msgstr ""

#: apps/search_providers/service.py:69
msgid "Deleting a Search Provider after receiving items is prohibited."
msgstr ""

#: apps/suggestions/service.py:34
msgid "The item identifier is required"
msgstr ""

#: apps/suggestions/service.py:39
msgid "Invalid item identifer"
msgstr ""

#: apps/templates/content_templates.py:183
msgid "Invalid kill template. {fields} are not allowed"
msgstr ""

#: apps/templates/content_templates.py:233
msgid "Kill templates can not be deleted."
msgstr ""

#: apps/templates/content_templates.py:321
msgid "Kill templates must be public"
msgstr ""

#: apps/templates/content_templates.py:333
msgid "Templates that are not create type can only be assigned to one desk!"
msgstr ""

